packages/@aws-cdk/aws-ecs/test/fargate/test.fargate-task-definition.ts

```diff
@@ -111,6 +111,29 @@ export = {
         taskDefinition.addPlacementConstraint(ecs.PlacementConstraint.memberOf("attribute:ecs.instance-type =~ t2.*"));
       }, /Cannot set placement constraints on tasks that run on Fargate/);
 
+      test.done();
+    },
+
+    'throws when setting proxyConfiguration without networkMode AWS_VPC'(test: Test) {
```

Comment:
Love the test, if you can move it to test.ec2-task-definition.ts that would be great!